pom.xml

```diff
@@ -253,7 +253,7 @@
         <swagger-parser-v2-version>1.0.38-SNAPSHOT</swagger-parser-v2-version>
```

Comment:
Having a snapshot in the dependencies is not a good idea in my opinion. Release should only have released versions in their dependencies. 

Otherwise, builds can break (if the `SNAPSHOT` version is remove or if it evolves to something that is no longer compatible)